lib/stat_tracker.rb

```diff
@@ -20,19 +20,76 @@ def self.from_csv(locations)
 #
 # #------------GameStatistics
 #
-#   def highest_total_score
-#     result = games.max_by do |game|
-#       game.away_goals.to_i + game.home_goals.to_i
-#     end
-#     result.away_goals.to_i + result.home_goals.to_i
-#   end
+  def highest_total_score
+    result = games.max_by do |game|
+      game['away_goals'].to_i + game['home_goals'].to_i
+    end
+    result['away_goals'].to_i + result['home_goals'].to_i
+  end
 #
-#   def lowest_total_score
-#     result = games.min_by do |game|
-#       game.away_goals.to_i + game.home_goals.to_i
-#     end
-#     result.away_goals.to_i + result.home_goals.to_i
-#   end
+  def lowest_total_score
+    result = games.min_by do |game|
+      game['away_goals'].to_i + game['home_goals'].to_i
+    end
+    result['away_goals'].to_i + result['home_goals'].to_i
+  end
+
+  def home_win_percentage
+    average = all_home_game_wins/all_home_games.count.to_f
```

Comment:
I agree with @Stearnzy! I like all the small methods which makes it easy to see what's going on

I would just clean up some spacing stuff by adding in a space before and after the `/`